api/src/main/java/org/pmiops/workbench/cdr/CdrVersionMapper.java

```diff
@@ -0,0 +1,23 @@
+package org.pmiops.workbench.cdr;
+
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.pmiops.workbench.db.model.DbCdrVersion;
+import org.pmiops.workbench.model.CdrVersion;
+import org.pmiops.workbench.utils.mappers.CommonMappers;
+import org.pmiops.workbench.utils.mappers.MapStructConfig;
+
+@Mapper(
+    config = MapStructConfig.class,
+    uses = {CommonMappers.class})
+public interface CdrVersionMapper {
+
+  @Mapping(source = "microarrayBigqueryDataset", target = "hasMicroarrayData")
+  @Mapping(source = "dataAccessLevelEnum", target = "dataAccessLevel")
+  @Mapping(source = "archivalStatusEnum", target = "archivalStatus")
+  CdrVersion dbModelToClient(DbCdrVersion db);
+
+  default boolean isNonNull(String s) {
+    return s != null;
```

Comment:
nit: null or empty might be slightly less error-prone. I'm worried about someone putting `microarrayBigQueryDataset: ""` in the JSON.